Python/bootstrap_hash.c

```diff
@@ -576,14 +581,15 @@ _Py_HashRandomization_Init(const PyConfig *config)
            pyurandom() is non-blocking mode (blocking=0): see the PEP 524. */
         res = pyurandom(secret, secret_size, 0, 0);
         if (res < 0) {
+            _Py_HashSecret_Initialized = 0;
```

Comment:
NOTE: An unrelated change, which I can remove... but I think it is right not to set this flag if initialization failed. Because then if you retried this, it would say OK with the secret still not being actually initialized.

Another way to do this "fix" is that I could move _Py_HashSecret_Initialized = 1 to where the _Py_InitNoneHash() is located, just before the return.

Reply:
Let's make this change a separate PR; it's either not needed, or is important enough to not be buried in this one.

Reply:
Reverted this, along with the hash-secret-dependent calculation.